### ISSUE TITLE
Multiline text loses space when displayed in html

### ISSUE THREAD

**Issue by rtoy** [tabatkins/bikeshed#1470]
For example, look at [numberOfChannels](https://webaudio.github.io/web-audio-api/#dom-baseaudiocontext-createbuffer-numberofchannels).  The text reads "Animplementation MUST..."  The actual bs source is:

>		numberOfChannels: Determines how many channels the buffer will have. An
>				  implementation MUST support at least 32 channels.

That is, the first line starts with two tabs and the third has three tabs. When rendered the two lines get smashed together with no space between the words.

The simple workaround is to make it one giant line.  This isn't great for editability/readability for long paragraphs.

Is this expected?

**Comment by rtoy** [comment]
Latest bikeshed updated just now still has this problem.  We'll apply the workaround in the next couple of days.

**Comment by rtoy** [comment]
Oops.  https://github.com/tabatkins/bikeshed/issues/1470#issuecomment-497509476 was meant for the web audio spec.

It's still true, but this issue is not a show-stopper since the workaround is fine for now.

**Comment by rtoy** [comment]
Interesting.  The [2018-06-19 draft](https://www.w3.org/TR/2018/WD-webaudio-20180619/#dom-baseaudiocontext-createbuffer) doesn't have this problem.

But it's there in the latest version.  Well, the workaround has been applied, so you'll need to look at a version before Jun 2 (https://github.com/WebAudio/web-audio-api/commit/099e144fb26efee9096d2095ab1f8818eb2b6bb0)

**Comment by tabatkins** [comment]
Yeah, workaround for now. It's somewhat confusing! If the argumentdef block is indented more than necessary, the lines keep a space when they're merged; if it's up against the edge (or the edge of the current markdown indent level), they don't have a space. I'm not sure why this is!

I'll look into this more and get it fixed, but it's not as trivial as I thought.
